### DISCUSSION TITLE
umi打包失败

### DISCUSSION THREAD

**Discussion by zk0816** [umijs/umi#7857] [Q&A]
ERROR in ./node_modules/internmap/src/index.js 5:0-165
Module not found: Error: Can't resolve '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/assertThisInitialized' in '/Users/feewee005/blog-manage/node_modules/internmap/src'
Did you mean 'assertThisInitialized.js'?
BREAKING CHANGE: The request '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/assertThisInitialized' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./node_modules/d3-array/src/index.js 58:0-49 58:0-49 58:0-49
 @ ./node_modules/d3/src/index.js 1:0-25 1:0-25
 @ ./node_modules/mermaid/dist/mermaid.core.js
 @ ./node_modules/@bytemd/plugin-mermaid/dist/index.esm.js 94:49-67
 @ ./src/pages/articles/Article/subpages/ArticlePage/index.tsx 26:0-45 42:4-11
 @ ./src/.umi-production/core/routes.ts
 @ ./src/.umi-production/umi.ts 10:0-42 24:31-40

ERROR in ./node_modules/internmap/src/index.js 6:0-129
Module not found: Error: Can't resolve '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/get' in '/Users/feewee005/blog-manage/node_modules/internmap/src'
Did you mean 'get.js'?
BREAKING CHANGE: The request '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/get' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./node_modules/d3-array/src/index.js 58:0-49 58:0-49 58:0-49
 @ ./node_modules/d3/src/index.js 1:0-25 1:0-25
 @ ./node_modules/mermaid/dist/mermaid.core.js
 @ ./node_modules/@bytemd/plugin-mermaid/dist/index.esm.js 94:49-67
 @ ./src/pages/articles/Article/subpages/ArticlePage/index.tsx 26:0-45 42:4-11
 @ ./src/.umi-production/core/routes.ts
 @ ./src/.umi-production/umi.ts 10:0-42 24:31-40

ERROR in ./node_modules/internmap/src/index.js 7:0-151
Module not found: Error: Can't resolve '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/getPrototypeOf' in '/Users/feewee005/blog-manage/node_modules/internmap/src'
Did you mean 'getPrototypeOf.js'?
BREAKING CHANGE: The request '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/getPrototypeOf' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./node_modules/d3-array/src/index.js 58:0-49 58:0-49 58:0-49
 @ ./node_modules/d3/src/index.js 1:0-25 1:0-25
 @ ./node_modules/mermaid/dist/mermaid.core.js
 @ ./node_modules/@bytemd/plugin-mermaid/dist/index.esm.js 94:49-67
 @ ./src/pages/articles/Article/subpages/ArticlePage/index.tsx 26:0-45 42:4-11
 @ ./src/.umi-production/core/routes.ts
 @ ./src/.umi-production/umi.ts 10:0-42 24:31-40

ERROR in ./node_modules/internmap/src/index.js 8:0-139
Module not found: Error: Can't resolve '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/inherits' in '/Users/feewee005/blog-manage/node_modules/internmap/src'
Did you mean 'inherits.js'?
BREAKING CHANGE: The request '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/inherits' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./node_modules/d3-array/src/index.js 58:0-49 58:0-49 58:0-49
 @ ./node_modules/d3/src/index.js 1:0-25 1:0-25
 @ ./node_modules/mermaid/dist/mermaid.core.js
 @ ./node_modules/@bytemd/plugin-mermaid/dist/index.esm.js 94:49-67
 @ ./src/pages/articles/Article/subpages/ArticlePage/index.tsx 26:0-45 42:4-11
 @ ./src/.umi-production/core/routes.ts
 @ ./src/.umi-production/umi.ts 10:0-42 24:31-40

ERROR in ./node_modules/internmap/src/index.js 9:0-145
Module not found: Error: Can't resolve '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/createSuper' in '/Users/feewee005/blog-manage/node_modules/internmap/src'
Did you mean 'createSuper.js'?
BREAKING CHANGE: The request '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/createSuper' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./node_modules/d3-array/src/index.js 58:0-49 58:0-49 58:0-49
 @ ./node_modules/d3/src/index.js 1:0-25 1:0-25
 @ ./node_modules/mermaid/dist/mermaid.core.js
 @ ./node_modules/@bytemd/plugin-mermaid/dist/index.esm.js 94:49-67
 @ ./src/pages/articles/Article/subpages/ArticlePage/index.tsx 26:0-45 42:4-11
 @ ./src/.umi-production/core/routes.ts
 @ ./src/.umi-production/umi.ts 10:0-42 24:31-40

ERROR in ./node_modules/internmap/src/index.js 10:0-153
Module not found: Error: Can't resolve '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/wrapNativeSuper' in '/Users/feewee005/blog-manage/node_modules/internmap/src'
Did you mean 'wrapNativeSuper.js'?
BREAKING CHANGE: The request '/Users/feewee005/blog-manage/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/wrapNativeSuper' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./node_modules/d3-array/src/index.js 58:0-49 58:0-49 58:0-49
 @ ./node_modules/d3/src/index.js 1:0-25 1:0-25
 @ ./node_modules/mermaid/dist/mermaid.core.js
 @ ./node_modules/@bytemd/plugin-mermaid/dist/index.esm.js 94:49-67
 @ ./src/pages/articles/Article/subpages/ArticlePage/index.tsx 26:0-45 42:4-11
 @ ./src/.umi-production/core/routes.ts
 @ ./src/.umi-production/umi.ts 10:0-42 24:31-40

webpack compiled with 87 errors
null
build failed
Error: build failed
    at /Users/feewee005/blog-manage/node_modules/@umijs/bundler-webpack/lib/index.js:172:29
    at /Users/feewee005/blog-manage/node_modules/@umijs/deps/compiled/webpack/5/bundle5.js:114591:14
    at processQueueWorker (/Users/feewee005/blog-manage/node_modules/@umijs/deps/compiled/webpack/5/bundle5.js:114529:6)
    at processTicksAndRejections (internal/process/task_queues.js:77:11)
    

antd pro 4 run start 正常 build时候 出现大辆这种现象  如何解决了

**Comment by Kujo-Jotaro-AAAAA** [comment]
+ 1  有类似报错

**Comment by cwjTerrace** [comment]
我这也出现了

**Comment by wildcat-var** [comment]
我也出现了，而且start的时候就已经出现了

**Reply by GerryIsWarrior** [reply]
解决了么？

**Comment by GerryIsWarrior** [comment]
解决了么？
